scripts/drtprod

```diff
@@ -456,6 +456,143 @@ WantedBy=multi-user.target
+          --local-ssd="true"  \
+          --gce-local-ssd-count="2" \
+          --username="drt" \
+          --lifetime="8760h"
+
+        roachprod create drt-ldr2 \
+          --clouds="gce" \
+          --gce-zones="us-east1-c" \
+          --nodes="5" \
+          --gce-machine-type="n2-standard-8" \
+          --local-ssd="true"  \
+          --gce-local-ssd-count="2" \
+          --username="drt" \
+          --lifetime="8760h"
+
+        # setup dns
+        $0 dns drt-ldr1 create || $0 dns drt-ldr1 || (echo "dns setup failed -- run dns manually"; true)
+        $0 dns drt-ldr2 create || $0 dns drt-ldr2 || (echo "dns setup failed -- run dns manually"; true)
+
+        # start cockroach
+        $0 stage drt-ldr1 cockroach
+        $0 stage drt-ldr2 cockroach
+        $0 start drt-ldr1 --restart=false
+        $0 start drt-ldr2 --restart=false
+
+        $0 sql drt-ldr1:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+        $0 sql drt-ldr2:1 -- -e "SET CLUSTER SETTING kv.rangefeed.enabled = true"
+
+        # import the workload
+        $0 sql drt-ldr1:1 -- -e "CREATE DATABASE kv"
+        $0 sql drt-ldr2:1 -- -e "CREATE DATABASE kv"
+        
+        $0 ssh drt-ldr1:1 "./cockroach workload init kv --insert-count=1000 {pgurl:1}"
+        $0 ssh drt-ldr2:1 "./cockroach workload init kv --insert-count=1000 {pgurl:1}"
+
+        # tell the clusters about eachother.
+        ldr2="$(roachprod ssh drt-ldr2:1 -- ./cockroach encode-uri --inline --ca-cert ./certs/ca.crt --key ./certs/client.root.key --cert ./certs/client.root.crt {pgurl:1:system} | sed 's,postgresql://postgres://,postgres://,')"
+        ldr1="$(roachprod ssh drt-ldr1:1 -- ./cockroach encode-uri --inline --ca-cert ./certs/ca.crt --key ./certs/client.root.key --cert ./certs/client.root.crt {pgurl:1:system} | sed 's,postgresql://postgres://,postgres://,')"
+        $0 sql drt-ldr1:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr2' AS '${ldr2}'"
+        $0 sql drt-ldr2:1 -- -e "CREATE EXTERNAL CONNECTION 'drt-ldr1' AS '${ldr1}'"
+        roachprod sql drt-ldr1:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE kv.kv ON 'external://drt-ldr2' INTO TABLE kv.public.kv;"
+        roachprod sql drt-ldr2:1 -- -e "CREATE LOGICAL REPLICATION STREAM FROM TABLE kv.kv ON 'external://drt-ldr1' INTO TABLE kv.public.kv;"
+
+        echo
+        echo "Completed setup of ldr1 and ldr2, with logical data replication between them on table kv.kv."
+        echo
+        echo "Run 'drtprod create workload-ldr' after the restore completes to start the workload."
+        echo
+        ;;
+      "workload-ldr")
+        set -x
+        roachprod create workload-ldr \
+          --clouds="gce" \
+          --gce-zones="us-east1-c" \
+          --nodes="1" \
+          --gce-machine-type="n2-standard-8" \
+          --os-volume-size 100 \
+          --username workload \
+          --lifetime 8760h \
+          --gce-image "ubuntu-2204-jammy-v20240319"
```

Comment:
Does it make sense to hard-code the image to use in the script?  Could we not instead use roachprod's latest image?

Reply:
Copied this from ua. But yeah it makes sense to not hardcode it and use the default.